### ISSUE TITLE
Token encoding exception for verify_bytes in Winzip (-m13600)

### ISSUE THREAD

**Issue by juanmera** [hashcat/hashcat#2200]
I got this error, even in benchmark mode:
```
$ hashcat -w3 -b -m13600
hashcat (v5.1.0-1403-g87022919+) starting in benchmark mode...

nvmlDeviceGetFanSpeed(): Not Supported

CUDA API (CUDA 10.1)
====================
* Device #1: GeForce GTX 860M, 2004 MB, 5MCU

OpenCL API (OpenCL 1.2 CUDA 10.1.0) - Platform #1 [NVIDIA Corporation]
======================================================================
* Device #2: GeForce GTX 860M, skipped

Benchmark relevant options:
===========================
* --workload-profile=3

Self-test hash parsing error: Token encoding exception

Started: Sat Oct 12 14:47:24 2019
Stopped: Sat Oct 12 14:47:25 2019
```

I tracked the problem to https://github.com/hashcat/hashcat/blob/master/src/modules/module_13600.c#L159 since the verify_bytes is a hex string, both in the benchmark and the result of zip2john.

**Comment by juanmera** [comment]
It seems that the next field, compressed_length should also be a hex string but in the benchmark is 0 so maybe is not needed.

**Comment by floyd-fuh** [comment]
Is it possible that https://github.com/floyd-fuh/JKS-private-key-cracker-hashcat/issues/3 is a similar issue and the module needs to be fixed? Should I open a new issue?

**Comment by jsteube** [comment]
Yes, fixed with commit 9a2c4e341782e347e5b3ebb0bb81b712593aafbd
